package.json

```diff
@@ -91,7 +92,7 @@
         "fix-lint": "run-s common-fix-lint backend-fix-lint frontend-fix-lint e2e-fix-lint cli-fix-lint",
         "fix-format": "run-s common-fix-format backend-fix-format frontend-fix-format e2e-fix-format cli-fix-format",
         "test": "run-s common-test warehouses-test cli-test backend-test frontend-test",
-        "dev": "npm-run-all --parallel common-dev warehouses-dev backend-dev frontend-dev",
+        "dev": "concurrently -n \"common,warehouses,backend,frontend\" --prefix-colors \"bgWhite.bold,bgGreenBright.bold,bgMagenta.bold,bgBlue.bold\" \"yarn:common-dev\" \"yarn:warehouses-dev\" \"yarn:backend-dev\" \"yarn:frontend-dev\"",
```

Comment:
The colors are really nice. 🎉 
<img width="574" alt="Screenshot 2023-07-25 at 16 16 39" src="https://github.com/lightdash/lightdash/assets/9117144/67c70380-e301-45f7-ab61-be7a0f8ba297">